public/index.html

```diff
@@ -13,7 +14,37 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>Kartika's Portfolio: Frontend Developer</title>
+    <!-- Meta tags for facebook -->
+    <meta property="og:title" content="Kartika's Portfolio" />
+    <meta property="og:url" content="https://kartikap.netlify.app/" />
+    <meta property="og:image" content="https://kartikap.netlify.app/images/ogimage.png" />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="og:description" content="Kartika, an aspiring frontend developer. Here you can find her projects she has done throughout her career as a web developer." />
+    <meta property="og:locale" content="en" />
+
+    <!-- OG:Image for Whatsapp or social media that needs smaller image -->
+    <meta property="og:image" content="https://kartikap.netlify.app/images/ogimage.png" />
+    <meta property="og:image:width" content="400" />
+    <meta property="og:image:height" content="400" />
+
+    <!-- Meta tags for twitter -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="twitter:domain" content="kartikap.netlify.app" />
+    <meta property="twitter:url" content="https://kartikap.netlify.app/" />
+    <meta name="twitter:title" content="Kartika's Portfolio" />
+    <meta name="twitter:description" content="Kartika, an aspiring frontend developer. Here you can find her projects she has done throughout her career as a web developer." />
+    <meta name="twitter:image" content="https://kartikap.netlify.app/images/ogimage.png" />
+
+    <!-- Icon on the tab title -->
+    <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
+    <link rel="manifest" href="/images/site.webmanifest">
```

Comment:
Impressive! Did not know about this! The icon on the tab is making your site even more professional!